models/queries/contributors.js

```diff
@@ -0,0 +1,6 @@
+const contributorsQueries = {
+  contribute:
+    "INSERT INTO contributions (amount, status) VALUES ($1, $2) RETURNING *"
```

Comment:
am having fun dont mind me ! so what do you think about just adding the default in the table instead of passing everytime that the contribution is pending

Reply:
There is a PR implementing that